test/verify/check-machines-dbus

```diff
@@ -819,6 +830,17 @@ class TestMachinesDBus(machineslib.TestMachines):
                 source={"host": "127.0.0.1", "source_path": target_iqn}
             ).execute()
 
+            # iscsi-direct pool type is available since libvirt 4.7 which is not available yet in RHEL
+            if "fedora" not in m.image:
```

Comment:
This looks wrong. Didn't you mean to say something like `if m.image not in ["rhel-7.6", "rhel-7.6-distropkg", "rhel-7.7", "centos-7"]:`?

Reply:
@KKoukiou It looks not orginally added by me. Do you know the reason why the code is added here?

Reply:
That's my fault. I wanted to write if fedora in m.image since should will be finally executed only on fedora-* images.  

All of the rhel-7/centos dont have yet libvirt-dbus so this is not going to run anyway.

I need to exclude ["rhel-8.0", "rhel-8.1"]
Sorry for getting it wron g here.